File: Test-Initialize/Test-Initialize/ViewController.m
//
//  ViewController.m
//  Test-Initialize
//
//  Created by Chill on 2024/4/10.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

+ (void)initialize {
    NSLog(@"ViewController -- initialize");
}

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    NSLog(@"%s", __func__);
}


@end

File: Test-Swift-Dispatch/Test-Swift-Dispatch/ViewController.swift
//
//  ViewController.swift
//  Test-Swift-Dispatch
//
//  Created by Chill on 2024/4/15.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        let test = TestProtocol()
        test.test1()
        
        SVG().render()
    }


}

